ts-client/src/dlmm/index.ts

```diff
@@ -1724,18 +1726,32 @@ export class DLMM {
       (pubkey) => new PublicKey(pubkey)
     );
 
-    const lbPairAndBinArrays = await chunkedGetMultipleAccountInfos(
+    const positionPubKeys = [...positions, ...positionsV2].map(
+      ({ publicKey }) => publicKey
+    );
+    const binArrayPubKeys = [...binArrayPubkeyArray, ...binArrayPubkeyArrayV2];
+
+    const lbPairAndBinArraysAndPositions = await chunkedGetMultipleAccountInfos(
```

Comment:
We need to make sure positions and bin arrays fetched in 1 RPC. The RPC limit is 100 account. Therefore, we need to make sure that the `[this.pubkey, SYSVAR_CLOCK_PUBKEY, ...binArrayPubKeys, ...positionPubKeys]` length for each RPC call is <= 100.